### ISSUE TITLE
Can't compile

### ISSUE THREAD

**Issue by ioncodes** [JonasFabian/Project1#2]
IntelliJ says "The solution could not be found". Any help on that?

**Comment by JonasFabian** [comment]
Yes. You should get a rope, then search a tree you can climb on. The next step i going to be very difficult, so watch out. You need to throw the rope over a branch of the tree. Then take the other end and make knob. There will be a ribbon, which you should put around your neck. After that you need to climb up the tree and jump down, like you never jumped before. I hope i helped you ;)
